### PR TITLE
New awesome feature built by TT

### PR DESCRIPTION
**Pull request by tartieret**
I did this and this and this

### CODE REVIEW

File: message.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Message from TT")
```
Comment (mk-panevo) [inline message.py:1]: This is great

Comment (mk-panevo) [inline message.py:1]: This should be better -- too specific from sender

**Review by mk-panevo — CHANGES_REQUESTED** [review]
See comments -- otherwise good

File: message.py (hunk 1 above)
Comment (tartieret) [reply message.py:1]: good idea

File: message.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("Better message from TT")
```
Comment (mk-panevo) [inline message.py:1]: Bravo!

**Review by mk-panevo — APPROVED** [review]
Good

**Pull request merged**